applications/crossbar/src/modules_v2/cb_users_v2.erl

```diff
@@ -233,7 +234,9 @@ validate(Context, UserId, ?QUICKCALL, _) ->
 -spec post(cb_context:context(), path_token()) -> cb_context:context().
 post(Context, _) ->
     _ = crossbar_util:maybe_refresh_fs_xml('user', Context),
-    crossbar_doc:save(Context).
+    Context1 = crossbar_doc:save(Context),
+    _  = maybe_update_devices_presence(Context),
```

Comment:
Shouldn't you test `cb_context:resp_status(Context1)` here for success. No need to update if the save failed!